server/routes/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2018 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an 'AS IS' BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+const express = require('express');
+const log4js = require('log4js');
+const config = require('config');
+
+const health = require('./health');
+
+const router = express.Router();
+
+/**
+ * Set up logging
+ */
+const logger = log4js.getLogger('routes - index');
+logger.setLevel(config.logLevel);
+
+/**
+ * Add routes
+ */
+router.use('/health', health);
+
+/**
+ * GET home page
+ */
+router.get('/', (req, res) => {
```

Comment:
Does there need to be a 'home page' for an API? Redirecting to the api-docs might be more useful?

Reply:
That's a great suggestion. I don't see a need for a home page, just something I threw in there